apps/web/src/pages/CommunityInterests/sections/AdditionalInformation.tsx

```diff
@@ -119,23 +119,14 @@ const AdditionalInformation = ({
   } as const;
 
   return (
-    <div
-      data-h2-display="base(flex)"
-      data-h2-flex-direction="base(column)"
-      data-h2-gap="base(x1.25)"
-    >
+    <div className="flex flex-col gap-7.5">
       {/* heading and description */}
-      <div
-        data-h2-display="base(flex)"
-        data-h2-flex-direction="base(column)"
-        data-h2-gap="base(x1)"
-      >
+      <div className="flex flex-col gap-6">
         <Heading
           level="h2"
-          data-h2-font-weight="base(400)"
           icon={ClipboardDocumentCheckIcon}
           color="secondary"
-          data-h2-margin="base(0)"
+          className="mt-0 font-normal"
```

Comment:
I've seen this in a couple places (but not all) in the PR.  This is intentional?

```suggestion
          className="m-0 font-normal"
```

Reply:
Yeah, exactly.